proto/data.proto

```diff
@@ -252,6 +252,22 @@ message Transaction {
   optional NodeList certain_nodes = 12 [(gogoproto.nullable) = false];
 }
 
+// Lease contains information about leader leases including the
+// expiration and lease holder.
+message Lease {
+  // The expiration is in unix nanos and is set according to the wall
+  // clock at the lease requestor / grantee.
+  optional int64 expiration = 1 [(gogoproto.nullable) = false];
+  // The leadership term for this lease.
+  optional uint64 term = 2 [(gogoproto.nullable) = false];
+  // The node ID of replica holding the lease.
```

Comment:
Well the RaftNodeID would work, as it's just the NodeID and StoreID. But we can't send the RaftID, as it only identifies a group, not the replica in the group. We must know the replica in the group to know whether it's us that has the lease or a different replica.